README.md

```diff
@@ -127,6 +127,63 @@ self.myColorSmallLabel.msLabel(MSFontStyle.XS, textColor: UIColor.msThemeSeconda
 
 ![Image showing large, medium, and small UI labels in several colors](https://raw.githubusercontent.com/OfficeDev/Office-UI-Fabric-iOS/master/DocsAssets/LabelExample.png)
 
+###UITextField
+
+You can use `UITextFieldMSExtension` to customize `UITextField`s in your project.
+
+![Image showing customized text fields with box and underline styles](https://raw.githubusercontent.com/OfficeDev/Office-UI-Fabric-iOS/master/DocsAssets/TextFieldExample.png)
+
+`UITextField` customization consist of three components:
+
+* TextField Form
+* Text Style
+* Placeholder Text Style
+
+####1. Define the form:
+
+You can use `Box` or `Underline` style for your text field:
+```swift
+self.myTextField.msTextFieldBox()
```

Comment:
@chedon I think we're not ready for this yet, because currently default values could change over time... Later we'll need something better for documentation (website section?)